db/db_local/backup.go

```diff
@@ -30,6 +35,10 @@ func prepareBackup(ctx context.Context) (*string, error) {
 	if !needs {
 		return nil, nil
 	}
+	// short circuit if there is an instance running
```

Comment:
// fail if there is a db instance running

Reply:
done